### PR TITLE
S2 game screen

### PR DESCRIPTION
**Pull request by Parsley-Afroman**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            </div>
+        </div>
+    </nav>
+    <main>
+        <div class="title"><h1>W&#129299RD NERD</h1></div>
+        <div class="letterTiles">
+
+        </div>
+        <div class="wordInput Input Outcome">
+            <form method="post" action="">
```
Comment (charliecog) [inline index.html:43]: action is empty - preferable to leave the attribute off altogther

File: index.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                <input type="text" placeholder="Guess Word ..." class="textInput">
+                <input type="submit" value="Enter" class="enter submitFormButton">
+                <input type="submit" value="Retry" class="retry retryButton">
+            </form>
+        </div>
+        <section class="keypadContainer">
+            <div class="keypadRow">
+                <button class="keypadKey" data-letter="Q">Q</button>
```
Comment (charliecog) [inline index.html:51]: might be worth making all the data-letters lower case to match the data you will be working with

File: index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const index = document.querySelector('.overlay')
+const modalClose = document.querySelector('.closeBtn')
+const instructionsBtn = document.getElementById('instructions')
+
+instructionsBtn.addEventListener('click', (e) => {
+    index.style.display = 'flex';
+    console.log('first')
```
Comment (charliecog) [inline index.js:7]: remove console.logs

File: index.html (hunk 1 above)
Comment (Parsley-Afroman) [reply index.html:43]: Done

File: index.html (hunk 2 above)
Comment (Parsley-Afroman) [reply index.html:51]: Done

File: index.js (hunk 3 above)
Comment (Parsley-Afroman) [reply index.js:7]: Done

**Review by charliecog — APPROVED** [review]

**Pull request merged**